client/src/views/Signup.tsx

```diff
@@ -37,39 +46,49 @@ const useStyles = makeStyles()((theme: Theme) => ({
 function Signup() {
   const { classes } = useStyles();
 
-  return (
-    <Container className={classes.formBox}>
-      <Typography variant="h2" component="h2">
-        Welcome!
-      </Typography>
-      <Typography variant="h2" component="h2">
-        Select your account type.
-      </Typography>
-      <Typography variant="body2" component="caption">
-        Already have an account? <Link to={routes.Login.path}>Log In</Link>{' '}
-      </Typography>
+  const [open, setOpen] = React.useState(true);
+  const onClose = (e: any, reason: string) => {
+    if (reason !== 'backdropClick') {
```

Comment:
I'm curious about `backdropClick` - I'll need to read up on this :)

Reply:
This is documented here: https://mui.com/material-ui/api/modal/#Modal-prop-onClose